pkgs/top-level/aliases.nix

```diff
@@ -92,6 +93,7 @@ mapAliases ({
   coprthr = throw "coprthr has been removed."; # added 2019-12-08
   corebird = throw "corebird was deprecated 2019-10-02: See https://www.patreon.com/posts/corebirds-future-18921328. Please use Cawbird as replacement.";
   coredumper = throw "coredumper has been removed: abandoned by upstream."; # added 2019-11-16
+  cpp_ethereum = throw "cpp_ethereum has been removed; abandoned upstream."; # added 2020-11-30
```

Comment:
```suggestion
  cpp_ethereum = throw "cpp_ethereum has been removed because upstream abandoned it and it is broken with current version of ethereum chain."; # added 2020-11-30
```

Reply:
It is not strictly broken *yet* - but likely will be by the time 21.03 is released once the staking updates for Ethereum go live next month